### PR TITLE
DO-1913_Add-RDS-Tags

### PR DESCRIPTION
**Pull request by david-trefou-ls**
Updated kitchen config file, fixed aws-sdk gem version, bumped cookbook version to 1.0.3

### CODE REVIEW

File: recipes/install.rb [hunk 1]
```diff
@@ -40,4 +40,9 @@
 end
 
 package 'ruby'
-%w(deep_merge mixlib-cli aws-sdk).each { |pkg| gem_package pkg }
+%w(deep_merge mixlib-cli).each { |pkg| gem_package pkg }
+
+gem_package 'aws-sdk' do
+  action :upgrade
+  version "#{node['castor']['aws-sdk-version']}"
```
Comment (ls-avanier) [inline recipes/install.rb]: Do you really need to string interpolate that?

**Comment by ls-avanier** [comment]
Generally seems legit except for that one line.
Love the `kitchen.yaml` being brought to the 21st century.

File: recipes/install.rb (hunk 1 above)
Comment (mathieumartin-ls) [inline recipes/install.rb]: Agreed. Please don't interpolate when there's no need.

```ruby
gem_package 'aws-sdk' do
  action :upgrade
  version node['castor']['aws-sdk-version']
```

**Comment by mathieumartin-ls** [comment]
Please always prefix your PR title with the ticket ID.

File: recipes/install.rb (hunk 1 above)
Comment (ls-jftheroux) [inline recipes/install.rb]: I would favor using :install and pinning the version instead, with whatever version you need. :upgrade leads to upgrades whenever something changes upstream. And could cause more problems.

**Pull request merged**